CHANGELOG.md

```diff
@@ -11,23 +11,37 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Badge to README to show code coverage percentage.
 - Support for Python 3.12.
-- Additional data classes `coreax.data.Data` and `coreax.data.SupervisedData` that draw
-distinction between supervised and unsupervised datasets, and handle weighted data.
+- Added deterministic, iterative, and greedy coreset algorithm which targets the Kernelized Stein Discrepancy
+via `coreax.solvers.coresubset.SteinThinning`.
+- Added a stochastic, iterative, and greedy coreset algorithm which approximates the Gramian of a given kernel function
+via `coreax.solvers.coresubset.RPCholesky`.
 - Added `coreax.util.sample_batch_indices` that allows one to sample an array of indices for batching.
-- Additional kernel classes `coreax.kernel.AdditiveKernel` and
-`coreax.kernel.ProductKernel` that allow for arbitrary composition of positive
-semi-definite kernels to produce new positive semi-definite kernels.
-- Added kernel classes `coreax.kernel.Linear`, `coreax.kernel.Polynomial`, `coreax.kernel.RationalQuadratic`,
- `coreax.kernel.Periodic`, `coreax.kernel.LocallyPeriodic`
-- Added capability to approximate regularised inverses of arrays `coreax.inverses.RegularisedInverseApproximator`
-- Added Conditional Maximum Mean Discrepancy (CMMD) metric in `coreax.metrics.CMMD`
+- Added kernel classes `coreax.kernel.AdditiveKernel` and `coreax.kernel.ProductKernel` that
+allow for arbitrary composition of positive semi-definite kernels to produce new positive semi-definite kernels.
+- Added additional kernel functions: `coreax.kernel.Linear`, `coreax.kernel.Polynomial`, `coreax.kernel.RationalQuadratic`,
+ `coreax.kernel.Periodic`, `coreax.kernel.LocallyPeriodic`.
+- Added capability to approximate the inverses of arrays via least-squares (`coreax.inverses.LeastSquaresApproximator`)
+or randomised eigendecomposition (`coreax.inverses.RandomisedEigendecompositionApproximator`) all inheriting
+from `coreax.inverses.RegularisedInverseApproximator`,
+- Refactor of package to a functional style to allow for JIT-compilation of the codebase in the largest possible scope:
```

Comment:
Need to mention use of Equinox now as an added dependency, which we now use instead of manually specifying pytrees

Reply:
Done.